### ISSUE TITLE
Spacemacs in Windows 10 (broken symbols)

### ISSUE THREAD

**Issue by dukebarman** [syl20bnr/spacemacs#12120]
#### Description :octocat:
Some symbols are broken in console mode

#### Reproduction guide :beetle:
- Start Emacs `C:\emacs\bin\emacs.exe -nw` in windows console or windows console through conemu

*Observed behaviour:* :eyes: :broken_heart:

![image](https://user-images.githubusercontent.com/1186777/55477117-076cef80-5621-11e9-9546-bd8b90fe4817.png)

*Expected behaviour:* :heart: :smile:

![image](https://user-images.githubusercontent.com/1186777/55476920-7bf35e80-5620-11e9-83c2-3268ee75c5ff.png)

#### System Info :computer:
- OS: windows-nt
- Emacs: 26.1
- Spacemacs: 0.200.13
- Spacemacs branch: nil (rev. nil)
- Graphic display: nil
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm emacs-lisp)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS THREADS LCMS2

**Comment by gcentauri** [comment]
@dukebarman, make sure your console is using a font that supports Unicode rendering. Is this just an issue with spacemacs, or do Unicode characters render otherwise?

**Comment by dukebarman** [comment]
I've tried:
- `chcp 65001` in terminal (to chane encoding)
- change font to Source Code Pro in conemu settings and windows's terminal

**Comment by tanevanwifferen** [comment]
Got this in linux too. behaviour started when I swiched to devel. Left is on my laptop, middle is over an ssh connection, right is an SSH tunnel from laptop -> server -> back to laptop.  Didn't change any fonts. 

![Screenshot from 2020-01-28 11-33-07](https://user-images.githubusercontent.com/6665486/73256424-277a2a80-41c2-11ea-996c-7481a222d129.png)


I found someone [here](https://gitter.im/syl20bnr/spacemacs/archives/2017/02/07?at=5899f8076b2d8dd55212d1e2) who had the same issues, to do with the LANG environment variable, but that is set correctly here.
